spec/spec_helper.rb

```diff
@@ -74,3 +74,14 @@
     end
   end
 end
+
+def get_message_part (mail, content_type)
+  mail.body.parts.find { |p| p.content_type.match content_type }.body.raw_source
+end
+
+shared_examples_for "multipart email" do
+  it "generates a multipart message (plain text and html)" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.